index.release.html

```diff
@@ -1,116 +1,140 @@
-        </dl></li>
-    </ul>
-  </div>
-  <script type="text/javascript">
-      if (window.location.protocol === 'file:') {
-          document.body.innerHTML = '';
-          document.write('<p><b>This file must be hosted in a web server.</br>');
-          document.write('See our <a href="https://cesiumjs.org/2013/04/12/Cesium-up-and-running/">Getting Started</a> ');
-          document.write('tutorial for a step-by-step guide.</b></p>');
-      }
-  </script>
+<body style="background-color: #f5f7f9;">
+<div>
+    <div class="section">
+        <a href="//cesiumjs.org"><img src="Apps/Sandcastle/images/Cesium_Logo_Color_Overlay.png" style='width: 400px'/></a>
+        <div style="font-weight: bold; margin: -20px 0 0 150px;">Next Dimension Mapping</div>
+    </div>
+    <div class="section">
+        <h2>Cesium ion</h2>
+        <p>
+            Cesium ion is your hub for discovering 3D content and tiling your own data for streaming. CesiumJS and ion
+            work together to enable you to build world class 3D mapping applications.
+        </p>
+        <p>
+            <a href="https://cesium.com/ion/signup/">Sign up for a free account</a> to get your access token required for using ion's Bing Maps global imagery and Cesium World Terrain assets.
+        </p>
+    </div>
+    <div class="section">
+        <h2>Local links</h2>
+        <table>
+            <tbody>
+            <tr>
+                <td><a href="Build/Documentation/index.html">Documentation</a></td>
+                <td>The complete API documentation and reference.</td>
+            </tr>
+            <tr>
+                <td><a href="Apps/HelloWorld.html">Hello World</a></td>
+                <td>The simplest possible Cesium application.</td>
+            </tr>
+            <tr>
+                <td><a href="Apps/CesiumViewer/index.html">Cesium Viewer</a></td>
+                <td>A sample Cesium reference application which allows you to browse the globe and select from a variety
+                    of imagery and terrain layers as well as load CZML, GeoJSON, and other formats supported by Cesium.
+                </td>
+            </tr>
+            <tr>
+                <td><a href="Apps/Sandcastle/index.html">Sandcastle</a></td>
+                <td>Cesium's live code editor and example gallery. Browse examples highlighting features of the Cesium
+                    API and edit and run them all in your web browser. Cesium applications created in Sandcastle can be
+                    saved and downloaded.
+                </td>
+            </tr>
+            <tr>
+                <td><a href="Specs/SpecRunner.html">Run tests</a></td>
+                <td>Run Cesium's full suite of unit tests</td>
+            </tr>
+            </tbody>
+        </table>
+    </div>
+    <div class="section">
+        <h2>External links</h2>
+        <table>
+            <tbody>
+            <tr>
+                <td><a href="//cesium.com">Cesium</a></td>
```

Comment:
I think all the leading `//` should change to `https://` at this point.

Reply:
👍 cool, I wasn't quite sure about that

Reply:
At some point many years ago, there was an effort to support users on both `http` and `https`, and preserve whichever protocol they're using.  But these days, best practice is to always upgrade to `https` anytime the server offers it.

Reply:
(which is not the case for webglreport, sadly, it's still http)